unittest/joint-configurations.cpp

```diff
@@ -46,6 +46,34 @@
 #include <boost/test/unit_test.hpp>
 
 
+Eigen::Quaterniond slerp(const Eigen::Quaterniond & q1, const Eigen::Quaterniond & q2,double t)
```

Comment:
And if you do not use Eigen, could you implement a SLERP interpolating the shortest path ?
`if(q1.dot(q2) < 0) mult2 = - mult2`